boards/arm/stm32l562e_dk/stm32l562e_dk.dts

```diff
@@ -48,3 +48,7 @@
 &iwdg {
 	status = "okay";
 };
+
```

Comment:
Would be nice to update board doc

Reply:
I have updated the board docs in a single commit.
As some other enabled peripherals were not listed as supported, I have added those as well.
Additionally I have ordered the features alphabetically, and added some links to the board configuration files.